sale_triple_discount/models/sale_order_line.py

```diff
@@ -39,6 +67,16 @@ def _compute_amount(self):
         digits=dp.get_precision('Discount'),
         default=0.0,
     )
+    discounting_type = fields.Selection(
+        string="Discounting type",
+        selection=[
+            ('additive', 'Additive'),
+            ('multiplicative', 'Multiplicative'),
+        ],
+        default="multiplicative",
+        help="""Specifies whether discounts should be added additively
```

Comment:
Don't put triple quota for this, or the help will be ugly, and also makes the help to be useful. What is `additively` and what `multiplicatively`